src/content/en/2021/caching.md

```diff
@@ -17,4 +19,141 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
```

Comment:
<p>Caching is a technique that enables the reuse of previously downloaded content. Over the past decade...</p>